### DISCUSSION TITLE
How to stop a task once it's started execution?

### DISCUSSION THREAD

**Discussion by iloveitaly** [Skyvern-AI/skyvern#393] [Q&A] [answered]
It doesn't look like there's a way to do this via the API or the UI. What am I missing?

**Comment by suchintan** [comment]
Hey! This is a great feature request. Right now we rely on MAX_STEPS_PER_RUN (https://github.com/Skyvern-AI/skyvern/blob/main/.env.example#L42) to timeout (instead of manually killing)

**Comment by suchintan** [comment, marked as answer]
@iloveitaly We just added support for this 

https://github.com/Skyvern-AI/skyvern/blob/6a6119b406b9c46adf51d763e2c64fea620b3e2f/skyvern/forge/sdk/routes/agent_protocol.py#L286
